includes/Plugin.php

```diff
@@ -239,6 +242,14 @@ public static function load( $main_file ) {
 			return false;
 		}
 
+		$config = new JSON_File( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'dist/config.json' );
+		Feature_Flags::set_instance(
+			new Feature_Flags(
+				new JSON_File( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'feature-flags.json' ),
+				$config['flagMode']
```

Comment:
Right now, this value may be empty (in production we won't ship the `config.json` file for now), which shouldn't be the case, as it would then bypass the `production` default and run the `googlesitekit_flag_mode` filter with an empty value. Let's make this more stable, we should pass `production` if `$config['flagMode']` is empty.